server/gokbg3/grails-app/domain/org/gokb/cred/TitleInstancePackagePlatform.groovy

```diff
@@ -417,7 +420,7 @@ class TitleInstancePackagePlatform extends KBComponent {
           }
 
           if (!cs_match) {
-            tipp.addToCoverageStatements('startVolume': c.startVolume, 'startIssue':c.startIssue, 'endVolume': c.endVolume, 'endIssue': c.endIssue, 'embargo':c.embargo, 'coverageNote': c.coverageNote, 'startDate': parsedStart, 'endDate': parsedEnd)
+            tipp.addToCoverageStatements('startVolume': c.startVolume, 'startIssue':c.startIssue, 'endVolume': c.endVolume, 'endIssue': c.endIssue, 'embargo':c.embargo, 'coverageDepth': c.coverageDepth, 'coverageNote': c.coverageNote, 'startDate': parsedStart, 'endDate': parsedEnd)
```

Comment:
This line creates a new TIPPCoverageStatement and adds it to the list of coverageStatements of the existing TIPP. As of right now, TIPPCoverageStatements do not have their own coverageDepth. There is an ongoing debate about wether a TIPP should be able to have statements of different depths or if this would constitute a new TIPP. For now, this change should be removed.

Reply:
if I understand correctly, there can be several coverage_data blocks per TIPP, as described in https://github.com/k-int/gokb-phase1/wiki/tipp_dto or is that the subject of debate, as you mentioned...?

Reply:
A TIPP can have multiple coverageStatements, yes. The debate is wether coverages of different depths should be included in the same TIPP, or if any TIPP should only ever have one 'global' depth (as it is right now).